README.md

```diff
@@ -77,7 +77,17 @@ More precisely:
 
 2. The Core Team will regularly review `swift-evolution` pull requests, and provide feedback to the authors in the pull request on whether or not the proposal looks within reason of something that might be accepted.
 
-3. If a proposal gets a positive indicator from the Core Team for later review, the authors must provide an implementation prior to the proposal being formally reviewed.  An implementation should be provided in the form of a pull request against the impacted repositories (e.g., `swift`, `swift-package-manager`), and the proposal should be updated with a link to that pull request.  The existence of an implementation does not guarantee that the proposal will be accepted, but it is instrumental in evaluating the quality and impact of the proposal.
+3. If a proposal gets a positive indicator from the Core Team for later review, the authors must provide an implementation prior to the proposal being formally reviewed.  An implementation should be provided in the form of a pull request against the impacted repositories (e.g., `swift`, `swift-package-manager`), and the proposal should be updated with a link to that pull request.  It is expected that before review begins the pull request implementation runs at least the following tests:
+
+    a. A full test on Linux and macOS. (e.x. "@swift-ci test").
+
+    b. A full test of the Source Compatibility Suite to identify source breakage. (e.x. "@swift-ci test source compatibility").
```

Comment:
I think we should elaborate on expectations.  For proposals that have no source-impacting changes, passing the Source Compatibility suite is generally expected.  For proposals that do expect source-impacting changes, they should quantify how the source compatibility suite is assessing expectations here.